qiskit/circuit/quantumcircuit.py

```diff
@@ -4400,7 +4400,7 @@ def if_test(
 
         There are two forms for calling this function.  If called with all its arguments (with the
         possible exception of ``label``), it will create a
-        :obj:`~qiskit.circuit.controlflow.IfElseOp` with the given ``true_body``, and there will be
+        :obj:`~qiskit.circuit.IfElseOp` with the given ``true_body``, and there will be
```

Comment:
If you wanted to change the `:obj:` everywhere, there's another here too.  I think for `:class:` it shouldn't make much of a difference, though.